### ISSUE TITLE
[Feature Request] Blocking mode for Machines with item in (like Circuits)

### ISSUE THREAD

**Issue by TheonlyTazz** [tom5454/LogisticsBridge#39]
Seeing how you added the Blocking mode to the buffer upgrade, may i suggest being able to change this so you can
have the Circuits in your Bender, Assembler etc? maybe make the amount of items customizable? 
![image](https://user-images.githubusercontent.com/3908847/89707177-1594f380-d96c-11ea-8393-05ff0ae2d0d4.png)

**Comment by dominic-jones** [comment]
Agreed. Perhaps this could be used to provide an exception for being considered empty. Items such as casts, moulds, configuration circuits etc. Items that by default stay in the machine and are used to modify the craft.
